share/picongpu/examples/Bunch/include/picongpu/param/incidentField.param

```diff
@@ -0,0 +1,238 @@
+
+                /** unit: none */
+                static constexpr float_64 _A0 = 1.0;
+
+                /** unit: Volt / meter */
+                static constexpr float_64 AMPLITUDE_SI = _A0 * UNITCONV_A0_to_Amplitude_SI;
+
+                /** Pulse length: sigma of std. gauss for intensity (E^2)
+                 *  PULSE_LENGTH_SI = FWHM_of_Intensity   / [ 2*sqrt{ 2* ln(2) } ]
+                 *                                          [    2.354820045     ]
+                 *  Info:             FWHM_of_Intensity = FWHM_Illumination
+                 *                      = what a experimentalist calls "pulse duration"
+                 *  unit: seconds (1 sigma)
+                 */
+                static constexpr float_64 PULSE_LENGTH_SI = 10.615e-15 / 4.0;
+
+                /** Laser phase shift (no shift: 0.0)
+                 *
+                 * sin(omega*time + laser_phase): starts with phase=0 at center --> E-field=0 at center
+                 *
+                 * unit: rad, periodic in 2*pi
+                 */
+                static constexpr float_X LASER_PHASE = 0.0_X;
+
+                /** Unit propagation direction vector
+                 *
+                 * Norm of this vector must be 1.0.
+                 * (Normalization is required on a user side as internally it is awkward to do with the
+                 * static-constexpr style of using parameters.)
+                 *
+                 * For 2d simulations, z component is not required, will be always set to 0.
+                 *
+                 * unit: none
+                 *
+                 * @{
+                 */
+                static constexpr float_64 DIRECTION_X = 0.0;
+                static constexpr float_64 DIRECTION_Y = 1.0;
+                static constexpr float_64 DIRECTION_Z = 0.0;
+                /** @} */
+
+                /** Focus position in total coordinate system
+                 *
+                 * "Non-focused" lasers should have it set at or near the generation surface where the laser enters
+                 * the domain. The position does not have to be inside the domain.
+                 *
+                 * The focus position and propagation direction together define the entry point of laser to
+                 * the generation surface.
+                 * So they also control the laser center at the generation plane, not just a shift inwards.
+                 *
+                 * For 2d simulations, z component has no effect and is not required.
+                 *
+                 * unit: m
+                 *
+                 * Note the plane wafe does not have a focus position therefore it is set to zero for all directions.
+                 *
+                 * @{
+                 */
+                static constexpr float_64 FOCUS_POSITION_X_SI = 0.0;
+                static constexpr float_64 FOCUS_POSITION_Y_SI = 0.0;
+                static constexpr float_64 FOCUS_POSITION_Z_SI = 0.0;

```

Comment:
Just a general question: why is there a focus position for the plane wave anyway? It has no meaning.